src/cargo/util/command_prelude.rs

```diff
@@ -1104,14 +1105,15 @@ pub fn get_registry_candidates(
     {
         Ok(registries
             .keys()
+            .sorted()
```

Comment:
- This appears to be unrelated to the current commit
- This should likely be a `fix` commit on its own